fwoa-tools/src/migrationVerify.test.ts

```diff
@@ -0,0 +1,144 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+jest.mock('./migrationUtils', () => ({
+    checkConfiguration: () => {},
+    getFhirClientSMART: () => {
+        //eslint-disable-next-line
+        return axios.create();
+    },
+    getFhirClient: () => {
+        //eslint-disable-next-line
+        return axios.create();
+    }
+}));
+
+import AWS from 'aws-sdk';
+import {GetObjectRequest} from "aws-sdk/clients/s3";
+import * as AWSMock from 'aws-sdk-mock';
+import axios from "axios";
+import MockAdapter from "axios-mock-adapter";
+import {logs} from "./migrationImport";
+import {parseCmdOptions, buildRunScriptParams, runScript, verifyResource} from "./migrationVerify";
+
+let mock: MockAdapter;
+const env = process.env;
+AWSMock.setSDKInstance(AWS);
+
+describe('migrationVerify', () => {
+    beforeAll(() => {
+        jest.spyOn(logs, 'write').mockImplementation((log: string) => {
+            console.log(log);
+            return true;
+        });
+        jest.spyOn(logs, 'end').mockImplementation(jest.fn());
+    });
+    beforeEach(() => {
+        mock = new MockAdapter(axios);
+        AWSMock.restore();
+    })
+    afterEach(() => {
+        mock.reset();
+        AWSMock.restore();
+        process.env = env;
+    })
+    describe('parseCmdOptions', () => {
+        test('smart and dryrun enabled', () => {
+            process.argv = [
+                '/usr/local/bin/ts-node',
+                'migrationVerify.ts',
+                '-s',
+                '-d',
+            ];
+
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            const argv: any = parseCmdOptions();
+            expect(argv.dryRun).toEqual(true);
+            expect(argv.smart).toEqual(true);
+        });
+        test('smart and dryrun not enabled', () => {
+            process.argv = [
+                '/usr/local/bin/ts-node',
+                'migrationVerify.ts'
+            ];
+
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            const argv: any = parseCmdOptions();
+            expect(argv.dryRun).toEqual(false);
+            expect(argv.smart).toEqual(false);
+        });
+    })
+    test ('buildRunScriptParams', () => {
+        process.argv = [
+            '/usr/local/bin/ts-node',
+            'migrationExport.ts'
+        ];
+        const {smartClient, dryRun} = buildRunScriptParams();
+        expect(dryRun).toEqual(false);
+        expect(smartClient).toEqual(false);
+    })
+
+    test('runScript', async() => {
+        process.env.EXPORT_BUCKET_NAME = 'fake-bucket-name'
+        const fakeFileBody =
```

Comment:
Nit: it'll be easier to read/update if we use `JSON.stringify(<the actual JSON object>)` here.

Reply:
updated